sdk/webpubsub/Azure.Messaging.WebPubSub/src/Generated/WebPubSubServiceClient.cs

```diff
@@ -38,6 +38,26 @@ protected WebPubSubServiceClient()
         /// <param name="role"> Roles that the connection with the generated token will have. </param>
         /// <param name="minutesToExpire"> The expire time of the generated token. </param>
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
+        /// <example>
+        /// This sample shows how to call GenerateClientTokenImplAsync with required parameters and properties.
```

Comment:
This example will be removed since there is no required parameter defined. We'll only generate the example for all paramter/request content, which will also solve the issue above.